applications/nrf5340_audio/include/nrf5340_audio_common.h

```diff
@@ -81,4 +81,11 @@ struct content_control_msg {
 	enum content_control_evt_type event;
 };
 
+/**
+ * @brief	Initialize the modules that are common for all the audio samples.
```

Comment:
```suggestion
 * @brief	Initialize the code modules that are common for all the audio samples.
```
- Or something else to make it clear that we are not talking about HW here.

Reply:
We are though, audio_system_init is part of this, which will init hw_codec

Reply:
Moved audio_system_init and changed doc slightly